redhat-cop/container-pipelines
# Basic-Nginx Removal of Input and Verify Gates

#### What does this PR do?
This PR removed the promotion input gate and verify deployment to the various stages in basic-nginx. The reasons being for a full autonomous demo, the input gates prevent this and the verify stages get stuck for hours when first deployed.

#### How should this be tested?
Existing testing should be sufficient, but confirmation that the user does not need to confirm promotion and verify there are no verify stages.

#### Is there a relevant Issue open for this?
No.

#### Who would you like to review this?
cc: @redhat-cop/containers-approvers
